src/com/amazon/ionelement/impl/ListElementImpl.kt

```diff
@@ -19,12 +19,13 @@ import com.amazon.ionelement.api.AnyElement
 import com.amazon.ionelement.api.ElementType
 import com.amazon.ionelement.api.ListElement
 import com.amazon.ionelement.api.MetaContainer
+import kotlinx.collections.immutable.toImmutableList
 
 internal class ListElementImpl (
     values: List<AnyElement>,
```

Comment:
What part do you think needs to be made private? If it's the constructor or the class, then it can't be instantiated by the function in `Ion.kt`. If it's `values`, that doesn't need to be private because it's not a member–it's just a function argument.

I can make them Persistent.

Reply:
> What part do you think needs to be made private? 

Nervermind, in haste I thought that was a property not a constructor  argument.